package.json

```diff
@@ -42,23 +42,28 @@
     "*.d.ts"
   ],
   "scripts": {
-    "chromatic": "chromatic -t 9b39ff142a7f",
     "build": "tsup",
     "build:staging": "CHROMATIC_BASE_URL=https://www.staging-chromatic.com tsup",
-    "build:watch": "pnpm run build --watch",
+    "build:watch": "run-p 'build --watch' 'codegen --watch'",
+    "build-storybook": "storybook build",
+    "chromatic": "chromatic -t 9b39ff142a7f",
+    "codegen": "graphql-codegen",
     "lint": "eslint src",
-    "test": "echo \"Error: no test specified\" && exit 1",
-    "start": "run-p build:watch 'storybook --quiet'",
     "release": "pnpm run build && auto shipit",
+    "start": "run-p build:watch 'storybook --quiet'",
     "storybook": "storybook dev -p 6006",
-    "build-storybook": "storybook build"
+    "test": "echo \"Error: no test specified\" && exit 1"
```

Comment:
Do we need the build-storybook command here?

Reply:
Yeah we should probably keep it for Chromatic.